go.mod

```diff
@@ -22,40 +21,13 @@ require (
 	github.com/sirupsen/logrus v1.6.0
 	github.com/subchord/go-sse v1.0.1
 	github.com/tecbot/gorocksdb v0.0.0-20191217155057-f0fad39f321c // indirect
-	github.com/tektoncd/pipeline v0.14.2
+	github.com/tektoncd/pipeline v0.16.3
 	github.com/unrolled/render v1.0.3
 	github.com/urfave/negroni/v2 v2.0.2
-	k8s.io/apimachinery v0.18.1
+	k8s.io/apimachinery v0.19.2
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
 )
 
-replace (
-	golang.org/x/sys => golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a // pinned to release-branch.go1.13
-	golang.org/x/tools => golang.org/x/tools v0.0.0-20190821162956-65e3620a7ae7 // pinned to release-branch.go1.13
-)
-
-replace k8s.io/api => k8s.io/api v0.16.5
-
-replace k8s.io/metrics => k8s.io/metrics v0.0.0-20190819143841-305e1cef1ab1
-
-replace k8s.io/apimachinery => k8s.io/apimachinery v0.16.5
-
-replace k8s.io/client-go => k8s.io/client-go v0.16.5
-
-replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190819143637-0dbe462fe92d
+replace k8s.io/client-go => k8s.io/client-go v0.19.2
```

Comment:
maybe we can change the "main" dependency version on client-go instead of replacing it here?

Reply:
I tried that - go seems to replace it automagically for some reason when you do a build ;) not totally sure why